frontend/src/types/Mesh.ts

```diff
@@ -16,3 +16,9 @@ export interface KialiInstance {
 }
 
 export type MeshClusters = MeshCluster[];
+
+// Internal Ambient Mesh indicators
+export const AmbientAnnotation = 'ambient.istio.io/redirection';
+export const AmbientAnnotationEnabled = 'enabled';
+export const WaypointLabel = 'gateway.istio.io/managed';
+export const WaypointLabelValue = 'istio.io-mesh-controller';
```

Comment:
That's right, those are duplicated, I added them in the public config to be sent from the server. 

I have some doubts about the right place to declare the constants/configs, I've moved the constants defined in `pod.go` to the config. 

I wonder, if something is in the config, it should be in the operator, right? And also, not sure if some of those values should be in the configuration to be able to be modified, as they are internal from Istio. I can see they could be valuable if for some reason those internal names changes in Istio, so it will be a workaround to set them correctly, but I cannot think of another reason.

Reply:
I guess there is no "right" answer.  If there is a constant that is used both on the server and the ui I guess it could be defined in both places.  But I guess it could also be defined on the server and passed to the ui via the serverConfig.  I don't think everything in config also needs to be customizable in the operator.  Maybe @jmazzitelli has input here.

Reply:
Thank you, @jshaughn 

I think it makes sense to define it just in one place, so you don't have to edit it twice (In case is needed), and avoid errors if both values are not the same...

Reply:
Even though "theoretically" these could change - will they really change in practice? I don't see a need to allow for these to be customizable in the Kiali config (like the configmap and the operator Kiali CR). These look pretty fixed to me. Defining them in one place is fine - but I would not add them to the configuration of Kiali - that just adds complexity to Kiali config where it doesn't need to be. If the Istio devs rename these for whatever reason, we'll update Kiali - I highly doubt these will be user-configurable, though.

Reply:
+1 @jmazzitelli , I don't think @josunect or I are thinking it will change.  I guess the question is only does it make sense to place these in Config.go, even ifthey are not configurable, and pass the values to the client, or is it better to define the constants in two places, the server and the client.  As is, these are defined once in Config.go and passed to the client.

Reply:
You can define them in Config.go - just don't add `"yaml:"` markers in the code - that will tell Go not to expect to see these in the ConfigMap.

Reply:
> +1 @jmazzitelli , I don't think @josunect or I are thinking it will change. I guess the question is only does it make sense to place these in Config.go, even ifthey are not configurable, and pass the values to the client, or is it better to define the constants in two places, the server and the client. As is, these are defined once in Config.go and passed to the client.

So maybe they can be defined in the config, but as constants, and use it in the `publicConfig`, like this? 

`https://github.com/kiali/kiali/pull/6627/files#diff-8362089f5ed4d46493d2e66f05eb1f3c499e1eee7dea42a563aeb13efe4b7aaeR77`

Reply:
That sounds like it would work OK.

Reply:
So, I've removed the `yaml:` in the `IstioLabels` struct for the Ambient labels: https://github.com/kiali/kiali/pull/6627/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R294 
I think it makes sense to have together all the Istio labels, even if they are from Ambient. 

If this does not suit, I would suggest to create a new struct with all the Ambient labels/annotations.

Reply:
Looks good.